src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/CompleteNextDataSetVersionImportFunctionTests.cs

```diff
@@ -88,6 +88,14 @@ public async Task Success()
 
             Assert.Equal(originalDataSetVersionImport.Id, updatedDataSetVersionImport.Id);
 
+            var updatedDataSetVersion = publicDataDbContext
+                .DataSetVersions
+                .Single(import => import.Id == nextVersion.Id);
+            
+            // Assert the updated data set version is set to status 'Finalising' by the `CompleteNextDataSetVersionImport` function.
```

Comment:
Not really an issue but I'm not sure that the "`by the CompleteNextDataSetVersionImport function.`" in the comment here is adding much, since this test is within the  CompleteNextDataSetVersionImportFunctionTests, so we can already assume it.